dpdata/orca/output.py

```diff
@@ -0,0 +1,64 @@
+from typing import Tuple
+
+import numpy as np
+
+
+def read_orca_sp_output(fn: str) -> Tuple[np.ndarray, np.ndarray, float, np.ndarray]:
+    """Read from ORCA output.
+
+    Note that both the energy and the gradient should be printed.
+
+    Parameters
+    ----------
+    fn : str
+        file name
+
+    Returns
+    -------
+    np.ndarray
+        atomic symbols
+    np.ndarray
+        atomic coordinates
+    float
+        total potential energy
+    np.ndarray
+        atomic forces
+    """
+    coord = None
+    symbols = None
+    forces = None
+    energy = None
+    with open(fn) as f:

```

Comment:
Consider handling the case where the file cannot be opened due to an IOError. This can be done by using a try-except block around the file opening operation. [important]